### DISCUSSION TITLE
cron connection refused using "Run an image that calls the cron job" method

### DISCUSSION THREAD

**Discussion by Solverz-0** [firefly-iii/firefly-iii#6549] [Firefly III] [answered]
Hello,

So the cronjob running inside the alpine container has the below error when it runs the daily wget:
`wget: can't connect to remote host (172.20.0.4): Connection refused`

`172.20.0.4` is the correct ip for the `app` container in the custom bridge docker network.

The command I am running in the `cron` container is: (`STATIC_CRON_TOKEN` is a placeholder for security posting in public)
`wget -qO- http://app/api/v1/cron/"STATIC_CRON_TOKEN"`

The alpine container name is `cron` and it is connected to a custom bridge network, which the `app` (firefly) container is also connected to.

`cron` can ping `app` perfectly fine.

The `wget -qO- http://app/api/v1/cron/"STATIC_CRON_TOKEN"` command works perfectly fine from another linux machine on the network, of course changing `app` to `ip:port`

Any ideas what I can try to find out what is wrong?

**Comment by JC5** [comment, marked as answer]
Add port 8080, so app:8080

**Reply by Solverz-0** [reply]
Immediatley after posting this I realised that **internally** in the firefly container it uses **port 8080** and not **port 80**.

So I just changed `http://app/api/v1/cron/"STATIC_CRON_TOKEN"` --> `http://app:8080/api/v1/cron/"STATIC_CRON_TOKEN"`

I guess I just assumed it would use port 80 internally for http and not a custom port.

The documentation section shows only `app` instead of `app:8080` in the example url, I think this should be changed to show the port too maybe.
> Run an image that calls the cron job

Thank you for the quick response and help!
